### DISCUSSION TITLE
what is the purpose of script patch - disable population load balancing?

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#3258] [Q&A]
https://github.com/YimMenu/YimMenu/blob/501182b10c4e63c57886e2d5a9589fa2af257278/src/backend/script_patches.hpp#L16

what does it do actually? Is it protection related or some feature related?

**Comment by Mr-X-GTA** [comment]
It was added to fix https://github.com/YimMenu/YimMenu/issues/734

**Reply by lonelybud** [reply]
its seems weird bcz in my fork i have removed this patch and I dont see any issues related to NPCs not being spawned on foot etc

**Reply by Rxann** [reply]
It might be obsolete as it was added quite a bit ago.
